helm-chart/kuberay-operator/values.yaml

```diff
@@ -27,7 +27,6 @@ resources:
   # resources, such as Minikube. If you do whelm to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
   limits:
-    cpu: 100m
```

Comment:
I think we should not remove the resource limit because the deployment will fail if the namespace has a `ResourceQuota` set up. How about increasing it a bit if it is not enough?